source/monitor/monitor_kraken/app.py

```diff
@@ -44,9 +44,6 @@
 
 
 def get_kraken_uri(instance_name):
-    uri = os.environ.get("KRAKEN_GENERAL_zmq_socket")
```

Comment:
Shouldn't we fallback to this in case config doesn't provide info? (which I guess is gonna be more common in the cloud?)
We may fallback before (no instance) also.

Not sure about all this, but this looks quite misleading (it is also probably not super important). Maybe just add a "deprecated" warning in code and in response?

Reply:
:heavy_check_mark: As discussed: doing nothing on that

Code comment does mention it's for multi-kraken monitoring, so it should be clear enough.

And fallback is not a big win to have a fallback on that... We may reconsider if it misleads somebody in the future, but it's unlikely.